compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/Scrollbar.desktop.kt

```diff
@@ -664,7 +678,33 @@ fun ScrollbarAdapter(
 @JvmName("ScrollbarAdapter2")
 fun ScrollbarAdapter(
     scrollState: LazyListState
-): androidx.compose.foundation.v2.ScrollbarAdapter = LazyScrollbarAdapter(scrollState)
+): androidx.compose.foundation.v2.ScrollbarAdapter = LazyListScrollbarAdapter(scrollState)
+
+/**
+ * ScrollbarAdapter for lazy grids.
+ *
+ * [scrollState] is instance of [LazyGridState] which is used by scrollable component
+ *
+ * Scrollbar size and position will be dynamically changed on the current visible content.
+ *
+ * Example:
+ *     Box(Modifier.fillMaxSize()) {
+ *         val state = rememberLazyGridState()
+ *
+ *         LazyVerticalGrid(state = state) {
+ *             ...
+ *         }
+ *
+ *         VerticalScrollbar(
+ *             Modifier.align(Alignment.CenterEnd),
+ *             rememberScrollbarAdapter(state)
+ *         )
+ *     }
+ */
+@JvmName("ScrollbarAdapter2")
+fun ScrollbarAdapter(
+    scrollState: LazyGridState
```

Comment:
This is a new function, so we don't need `@JvmName`

Reply:
I did it to be consistent with the other functions of the same kind. So that someone looking at the Java names (maybe in the debugger) has less to be confused about.

Reply:
Yes, good idea! Thanks